index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8" />
+        <title>The Weatherly</title>
+        <link href="./styles/index.css" rel="stylesheet" > 
+    </head>
+
+    <body>
+        <section>
+            <header>
+                <h1>The Weather For</h1>
+                <span id="display"> &#x1f3d9; Seattle &#x1f3d9; </span>
+            </header>
+        </section>
+        <main>
+            <div>
+                <section>
+                    <h3>Temperature</h3>
+                    <span id="temp_number" class="yellow">67</span>
```

Comment:
You should avoid class names like `yellow` and instead use something like `warm` or `hot`.

A class name should describe what the thing means, not its effect.